csrc/device_lower/validation.h

```diff
@@ -71,4 +71,7 @@ void validateLookupTV(Fusion* fusion);
 //! Check that there are no reductions over unexpanded broadcasts
 void validateReductions(Fusion* fusion);
 
+//! Validate f split output domain is loaded with 1D TMA, the split must be
```

Comment:
```suggestion
//! Validate if split output domain is loaded with 1D TMA, the split must be
```

Please add some context why this needs to hold.